src/Cell.cpp

```diff
@@ -91,11 +111,116 @@ cellType Cell::getType() const {
 
 
 /**
- * @brief Return the ID of the Universe within which this Cell resides.
- * @return the Universe ID
+ * @brief Return the minimum reachable x-coordinate in the Cell.
+ * @return the minimum x-coordinate
+ */
+double Cell::getMinX() {
+  return _min_x;
+}
+
+
+/**
+ * @brief Return the maximum reachable x-coordinate in the Cell.
+ * @return the maximum x-coordinate
+ */
+double Cell::getMaxX() {
+  return _max_x;
+}
+
+
+/**
+ * @brief Return the minimum reachable y-coordinate in the Cell.
+ * @return the minimum y-coordinate
+ */
+double Cell::getMinY() {
+  return _min_y;
+}
+
+
+/**
+ * @brief Return the maximum reachable y-coordinate in the Cell.
+ * @return the maximum y-coordinate
+ */
+double Cell::getMaxY() {
+  return _max_y;
+}
+
+
+/**
+ * @brief Return the minimum reachable z-coordinate in the Cell.
+ * @return the minimum z-coordinate
+ */
+double Cell::getMinZ() {
+  return _min_z;
+}
+
+
+/**
+ * @brief Return the maximum reachable z-coordinate in the Cell.
+ * @return the maximum z-coordinate
  */
-int Cell::getUniverseId() const {
-  return _universe;
+double Cell::getMaxZ() {
+  return _max_z;
+}
+
+
+/**
+ * @brief Return the boundary condition (REFLECTIVE, VACUUM, or INTERFACE) at
+ *        the minimum reachable x-coordinate in the Cell.
+ * @return the boundary condition at the minimum x-coordinate
+ */
+int Cell::getMinXBoundaryType() {
```

Comment:
Why the move from "left" to "minX"?  Are we ramping up for BCs on weird shaped cells?  Or are we just trying to sound more technical?